diff_drive_controller/src/diff_drive_controller.cpp

```diff
@@ -241,6 +245,8 @@ namespace diff_drive_controller{
 
     setOdomPubFields(root_nh, controller_nh);
 
+    cmd_vel_pub_.reset(new realtime_tools::RealtimePublisher<geometry_msgs::TwistStamped>(controller_nh, "cmd_vel_out", 100));
```

Comment:
@bmagyar @graiola So as of now the controller do instanciate the publisher, therefore the topic. This modification would allow the controller to instantiate only `if (publish_cmd_)`.
It is the reason the `default` test fails now.  
The question is then, do I include the modification or fix the test ? What do you think ?